staff/adrian-martinez/objects/index.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Objects</title>
+</head>
+<body>
+    <script src="reverseElements.js"></script>
+</body>
+</html>
```

Comment:
you should delete html file, we are working with node commands or console/inspector in the chrome

Reply:
Cierto, pero tampoco está de más, por si te falla node ;)